alot/settings/manager.py

```diff
@@ -311,8 +311,7 @@ def colourpick(triple):
     def get_hook(self, key):
         """return hook (`callable`) identified by `key`"""
         if self.hooks:
-            if key in self.hooks.__dict__:
-                return self.hooks.__dict__[key]
+            return getattr(self.hooks, key, None)
```

Comment:
Yup. I had to look to see. It's strange that getattr and dict.get behave differently given that getattr is effecitvely a wrapper around dict.get.